tests/unattended/unit/suites/test-elasticsearch.sh

```diff
@@ -167,55 +162,56 @@ test-configureElasticsearch-dist-two-elastic-nodes() {
     getConfig elasticsearch/roles/roles_mapping.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml
     getConfig elasticsearch/roles/internal_users.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml
 
     rm /etc/elasticsearch/esnode-key.pem /etc/elasticsearch/esnode.pem /etc/elasticsearch/kirk-key.pem /etc/elasticsearch/kirk.pem /etc/elasticsearch/root-ca.pem -f
-    mkdir /etc/elasticsearch/certs
+    copyCertificatesElasticsearch
 
     sed -i "s/-Xms1g/-Xms1g/" /etc/elasticsearch/jvm.options
     sed -i "s/-Xmx1g/-Xmx1g/" /etc/elasticsearch/jvm.options
 
-    applyLog4j2Mitigation
+    getConfig elasticsearch/elasticsearch_unattended_distributed.yml /etc/elasticsearch/elasticsearch.yml
 
-    copyCertificatesElasticsearch
+    applyLog4j2Mitigation
 
-    rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml -f
     /usr/share/elasticsearch/bin/elasticsearch-plugin remove opendistro-performance-analyzer
-}
-
-function load-configureElasticsearchAIO() {
-    @load_function "${base_dir}/elasticsearch.sh" configureElasticsearchAIO
+    rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml -f
 }
 
 test-configureElasticsearch-AIO() {
-    load-configureElasticsearchAIO
+    load-configureElasticsearch
+    AIO=1
     elasticsearch_node_names=("elastic1")
     elasticsearch_node_ips=("1.1.1.1")
     @mock free -g === @out "1"
     @mocktrue awk '/^Mem:/{print $2}'
-    configureElasticsearchAIO
+    @mock java -version === @out
+    @mock grep -o -m1 '1.8.0' === @out 1.8.0
+    configureElasticsearch
 }
 
 
 test-configureElasticsearch-AIO-assert() {
-    getConfig elasticsearch/elasticsearch_all_in_one.yml /etc/elasticsearch/elasticsearch.yml
     getConfig elasticsearch/roles/roles.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles.yml
     getConfig elasticsearch/roles/roles_mapping.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml
     getConfig elasticsearch/roles/internal_users.yml /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml
     
     export JAVA_HOME=/usr/share/elasticsearch/jdk/
     rm /etc/elasticsearch/esnode-key.pem /etc/elasticsearch/esnode.pem /etc/elasticsearch/kirk-key.pem /etc/elasticsearch/kirk.pem /etc/elasticsearch/root-ca.pem -f
 
-    mkdir /etc/elasticsearch/certs/
     copyCertificatesElasticsearch
 
     sed -i 's/-Xms1g/-Xms1g/' /etc/elasticsearch/jvm.options
     sed -i 's/-Xmx1g/-Xmx1g/' /etc/elasticsearch/jvm.options
 
+    getConfig elasticsearch/elasticsearch_all_in_one.yml /etc/elasticsearch/elasticsearch.yml
+
     applyLog4j2Mitigation
+
+    rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml -f
```

Comment:
You can use rm /etc/elasticsearch/{name1,name2,...}`, and please place the `-f` at the start

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1188/commits/107118c34312b821527a77cfc9dfbc175bcb14e4